### PR TITLE
added column_options

### LINKED ISSUE #10: Add column_options to TdOutputPlugin
Add column_options parameters to TdOutputPlugin to customize timestamp formatter.
https://github.com/embulk/embulk/pull/224

### PR DESCRIPTION
**Pull request by muga**
resolve #10

### CODE REVIEW

File: src/main/java/org/embulk/output/td/TdOutputPlugin.java [hunk 1]
```diff
@@ -96,8 +99,19 @@
...
-        @ConfigInject
-        public ScriptingContainer getJRuby();
+        @Override
+        @Config("default_timezone")
+        @ConfigDefault("\"UTC\"")
+        public DateTimeZone getDefaultTimeZone(); // default timezone
+
+        @Override
+        @Config("default_timestamp_format")
+        @ConfigDefault("\"%Y-%m-%d %H:%M:%S.%3N\"")
```
Comment (frsyuki) [inline src/main/java/org/embulk/output/td/TdOutputPlugin.java]: why don't you add comment here? To explain why default timestamp format is `%Y-%m-%d %H:%M:%S.%3N`

File: src/main/java/org/embulk/output/td/TdOutputPlugin.java [hunk 2]
```diff
@@ -96,8 +99,19 @@
         @ConfigDefault("16384") // default 16MB (unit: kb)
         public long getFileSplitSize();
 
-        @ConfigInject
-        public ScriptingContainer getJRuby();
+        @Override
+        @Config("default_timezone")
+        @ConfigDefault("\"UTC\"")
+        public DateTimeZone getDefaultTimeZone(); // default timezone
```
Comment (frsyuki) [inline src/main/java/org/embulk/output/td/TdOutputPlugin.java]: you don't need this override

Comment (muga) [inline src/main/java/org/embulk/output/td/TdOutputPlugin.java]: right. fixed d44292880a5615c8e76119d34b4d5bebbcea6b31

File: src/main/java/org/embulk/output/td/TdOutputPlugin.java (hunk 1 above)
Comment (muga) [inline src/main/java/org/embulk/output/td/TdOutputPlugin.java]: that's good. d44292880a5615c8e76119d34b4d5bebbcea6b31

**Pull request merged**